pkgs/tools/misc/bdfresize/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "bdfresize";
+  version = "1.5";
```

Comment:
For 20 years there was no new release. I'm wondering if there are no alternatives around which are maintained.

Reply:
I don't know about any alternatives. There are probably GUI programs around that provide all the functionality of bdfresize, but I'd like to resize fonts programmatically from within a Nix builder. If you can point me to a maintained alternative then I would be happy to package that instead.